cupyx/scipy/interpolate/__init__.py

```diff
@@ -3,3 +3,4 @@
 from cupyx.scipy.interpolate._polyint import KroghInterpolator  # NOQA
 from cupyx.scipy.interpolate._polyint import barycentric_interpolate  # NOQA
 from cupyx.scipy.interpolate._polyint import krogh_interpolate  # NOQA
+from cupyx.scipy.interpolate._rgi import RegularGridInterpolator,interpn
```

Comment:
Would you split this line to RegularGridInterpolator and interpn to follow our convention?

Reply:
Done